packages/paste-website/src/components/Heading.tsx

```diff
@@ -94,14 +96,40 @@ const StyledAnchorHyperlink = styled.a`
   }
 `;
 
+const StyledHeaderTag = styled.span`
+  margin-left: ${themeGet('space.space40')};
+  padding: ${themeGet('space.space10')} ${themeGet('space.space20')};
+  font-size: ${themeGet('fontSizes.fontSize20')};
+  font-weight: ${themeGet('fontWeights.fontWeightNormal')};
+  text-transform: uppercase;
```

Comment:
Generally, I'd like to avoid using `text-transform: uppercase` as a best practice. It doesn't translate well. 

Not a big deal for this site as I doubt we'll translate it to different languages, but I'd just like to make it a habit that we don't do it.

Reply:
+1, also not a fan of uppercase because it's less readable